cloud/scope/powervs_cluster.go

```diff
@@ -1105,8 +1105,27 @@ func (s *PowerVSClusterScope) ReconcileVPCSubnets() (bool, error) {
 		}
 		subnets = append(subnets, subnet)
 	}
-	for _, subnet := range subnets {
-		s.Info("Reconciling VPC subnet", "subnet", subnet)
+	powerVSZone := s.Zone()
+	if powerVSZone == nil {
+		return false, fmt.Errorf("powervs zone is not set")
+	}
+	region := endpoints.ConstructRegionFromZone(*powerVSZone)
+	vpcZones, err := genUtil.VPCZonesForPowerVSRegion(region)
+	if err != nil {
```

Comment:
I know its in WIP but came across this, You can now derive vpc zone from vpc region instead of Power VS zone, For reference https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/1819

Reply:
Yeah sure I am rebasing my PR with latest changes.